alifedata_phyloinformatics_convert/_aux/_all_unique.py

```diff
@@ -0,0 +1,15 @@
+import numpy as np
+
+from ._jit import jit
+
+
+@jit(nopython=True)
```

Comment:
Strip out the jit, because we don't already have jit dependencies and infrastructiure in the repo, I don't think the performance vs maintainability and installability tradeoff favors keeping jit.

Reply:
Done in 6760c0d